src/main/ColumnType.F90

```diff
@@ -115,6 +118,8 @@ subroutine Init(this, begc, endc)
     allocate(this%active      (begc:endc))                     ; this%active      (:)   = .false.
     allocate(this%type_is_dynamic(begc:endc))                  ; this%type_is_dynamic(:) = .false.
 
+    allocate(this%is_fates(begc:endc))                         ; this%is_fates(:) = .false.
```

Comment:
after discussion we decided to hold on changing this. We might change it later, but we also are not sure if we will move to make a fates filter instead of a logical mask anyway.